ntoskrnl/mm/ARM3/sysldr.c

```diff
@@ -2409,7 +2409,22 @@ MiSetSystemCodeProtection(
         TempPte = *PointerPte;
 
         /* Make sure it's valid */
+#if !defined(CORE_16449_IS_FIXED) && DBG
```

Comment:
Why special handling for DBG? Why not just always DPRINT1 and continue?

Reply:
(Because I often get complains about touching code/behavior I should not... That was before Thomas hijacked this part.)